src/versionbits.cpp

```diff
@@ -203,9 +203,9 @@ ThresholdState VersionBitsState(const CBlockIndex* pindexPrev, const Consensus::
     return VersionBitsConditionChecker(pos).GetStateFor(pindexPrev, params, cache.caches[pos]);
 }
 
-VBitsStats VersionBitsStatistics(const CBlockIndex* pindexPrev, const Consensus::Params& params, Consensus::DeploymentPos pos)
+VBitsStats VersionBitsStatistics(const CBlockIndex* pindex, const Consensus::Params& params, Consensus::DeploymentPos pos)
```

Comment:
Sorry, I think this change is wrong and it should be the other way around -- `AbstractThresholdConditionChecker::GetStateStatisticsFor` should be expecting `pindexPrev` not `pindex`. I believe that gives the right behaviour for MUST_SIGNAL. I'd suggest just dropping this commit.